ansible/roles/haproxy/templates/haproxy.cfg.j2

```diff
@@ -1329,9 +1329,6 @@ listen blazar_api_external
 listen prometheus_server
   bind {{ kolla_internal_vip_address }}:{{ prometheus_port }}
   http-request del-header X-Forwarded-Proto
-{% for http_option in haproxy_listen_http_extra %}
```

Comment:
I think we can drop the commit 941536a8c0 - it was part of a backport of Prometheus to Queens, and in Rocky these extra http options are available.

Reply:
Done.